components/UsersList/UsersList.tsx

```diff
@@ -49,34 +59,66 @@ export function UsersList({
     setRemovedUsername(user.username);
   }
 
+  const onSearchAction = (value: string) => {
+    if (!value.trim().length) {
```

Comment:
[TIL] Did not know that you could check just `.length` without doing a `.length() === 0` type of thing, that's cool

Reply:
Yes! In Javascript, `0` and `-0` are [falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy) whereas all other numbers are [truthy](https://developer.mozilla.org/en-US/docs/Glossary/Truthy), even negative ones (in this case, it's impossible to have a negative number). But I'll change this to `if (value.trim().length <= 0)` since the double negative is a bit confusing.